README.md

```diff
@@ -30,43 +30,145 @@ Software required for the proper functioning of the RSAN will be deployed on the
-puppetlabs/apt (>= 2.0.0 < 8.0.0)
-puppet-grafana (>= 3.0.0 < 7.0.0)
-puppet-telegraf (>= 2.0.0 < 4.0.0)
-puppetlabs-apt (>= 4.3.0 < 8.0.0)
-puppetlabs-inifile (>= 2.0.0 < 5.0.0)
-puppetlabs-puppetserver_gem (>= 1.1.1 < 3.0.0)
-puppet/openvpn (>= 8.3.0)
 
+Module Dependencies
 
+ - derdanne/nfs (>= 2.1.5)
+ - puppetlabs/postgresql (>= 6.6.0)
+ - puppetlabs/puppet_metrics_dashboard (>= 2.3.0)
+ - puppetlabs/stdlib (>= 4.5.0 < 7.0.0)
+- puppetlabs/concat (>= 1.1.2 < 7.0.0)
+- puppetlabs/transition (>= 0.1.0 < 1.0.0)
+- herculesteam/augeasproviders_core (>= 2.1.5 < 4.0.0)
+- herculesteam/augeasproviders_shellvar (>= 1.2.0 < 5.0.0)
+- puppetlabs/apt (>= 2.0.0 < 8.0.0)
+- puppet-grafana (>= 3.0.0 < 7.0.0)
+- puppet-telegraf (>= 2.0.0 < 4.0.0)
+- puppetlabs-apt (>= 4.3.0 < 8.0.0)
+- puppetlabs-inifile (>= 2.0.0 < 5.0.0)
+- puppetlabs-puppetserver_gem (>= 1.1.1 < 3.0.0)
 
 
 ### Beginning with rsan
 
-RSAN has Two Classes:
+RSAN has two main classes for use in the installation:
 
  - rsan::exporter - to be applied to all Puppet infrastructure agents - Console node group "PE Infrastructure Agent"
- - rsan::importer - to be applied to a single node which will be come the Remote Support Access Node
+ - rsan::importer - to be applied to a single node which will be come the Remote Support Access Node(RSAN)
+
+Following the application of these clases to the infrastructure, Puppet Will need to be run on the corresponding agents in the following order:
 
-Adding these two classes will set up all applications and configurations to run  RSAN
+Infrastructure Agent(s)->RSAN Agent->Infrastrcture Agent(s)->RSAN Agent
 
 ## Usage
+The following outlines the main features of RSAN and how to consume them
+### Live Telemetry Display
+<<<<<<< HEAD
+
+The Rsan node will host an instance of the [Puppet Metrics Dashboard](https://forge.puppet.com/modules/puppetlabs/puppet_metrics_dashboard)
+ 
+The Dashboard can be accessed on
+
+http://<RSAN-ip\>:3000\
+User: admin\
+Password: admin
+
+For advanced configuration and documentation please see [Puppet Metrics Dashboard](https://forge.puppet.com/modules/puppetlabs/puppet_metrics_dashboard)
+
+### Infrastructure node file and log access	
+
+The RSAN node will, by default, mount /var/log/ /opt/puppetlabs and /etc/puppetlabs from each of the Puppet Enterprise Infrastructure nodes on the RSAN platform in the following location, as read only file systems.
+
+/var/pesupport/<FQDN of Infrastructure node\>/var/log\
+/var/pesupport/<FQDN of Infrastructure node\>/opt/puppetlabs\
+/var/pesupport/<FQDN of Infrastructure node\>/etc/puppetlabs
+
+#### Optional Configuration
+
+The RSAN Class assumes the RSAN server will mount the shared partitions using the IP address Source designated by the "ipaddress" fact. In any deployment should this assertion not be true, it is nessary to set the following parameter to the source IP address of the RSAN Host:
+
+**rsan::exporter::rsan_importer_ips**
```

Comment:
This should show the syntax of setting this parameter or link to the REFERENCE.md for information.

In hiera"

```
rsan::exporter::rsan_importer_ips:
  - 1.2.3.4
 ```

In the class declaration or in the console the value would be an array

```
["1.2.3.4"]
```